test-suite-http-client-tck-jdk/src/test/java/io/micronaut/http/client/tck/jdk/tests/JdkHttpMethodTests.java

```diff
@@ -14,6 +14,7 @@
 @SuppressWarnings("java:S2187") // This runs a suite of tests, but has no tests of its own
 @ExcludeClassNamePatterns({
     "io.micronaut.http.client.tck.tests.ContinueTest", // Unsupported body type errors
+    "io.micronaut.http.client.tck.tests.ContentLengthHeaderTest", // GET requests add content-length header until https://github.com/openjdk/jdk/commit/6a7709320d28d8e1593b113fdf39ab583fca3687
```

Comment:
can we annotate the test with [@EnabledOnJre](https://junit.org/junit5/docs/5.2.0/api/org/junit/jupiter/api/condition/EnabledOnJre.html) or similar and run it in Java 21?

Reply:
Done in https://github.com/micronaut-projects/micronaut-core/pull/10218/commits/5ea08a89dcb77a93aedcfa7d238b113143438ea6